src/app/api/quizzes/[quizId]/questions/[questionNumber]/answers/route.ts

```diff
@@ -28,16 +28,16 @@ export async function GET(
 
     if (results.length === 0) {
       return NextResponse.json(
-        { error: quizMessages.notFound },
+        { error: quizMessages.notFoundQuestion },
         { status: 404 }
       );
     }
 
     const question = results[0];
 
     const [answerRows] = await db.query(
-      "SELECT id, answer, is_correct, question_id FROM Answer WHERE question_id = ?",
-      [question.id]
+      "SELECT a.id, a.answer, a.is_correct FROM Answer AS a JOIN Questions AS q ON a.question_id = q.id WHERE q.id = ? AND q.number = ?",
```

Comment:
you still have two requests, you can refactor by just this one, you don't need the question.id since you have the quiz id and the question number so you can select whatever you want from answer with a join on question where question.quiz_id = quizId and answer.question_number = questionNumber